### DISCUSSION TITLE
Inetutils

### DISCUSSION THREAD

**Discussion by Happy-root** [termux/termux-app#2784] [General]
When in want to use whois, prompt back install inetutils,but inetutils install
![Screenshot_20220522-101446](https://user-images.githubusercontent.com/106005080/169680723-f4054b35-5fed-468c-a4b0-ae2488ea89f7.png)

**Comment by ghost** [comment]
`whois` command is in `whois` package. (`pkg in whois`)

**Reply by Happy-root** [reply]
Yes
Thats work
Thanks :)
